docs/JestObjectAPI.md

```diff
@@ -568,6 +568,20 @@ jest.isolateModules(() => {
 const otherCopyOfMyModule = require('myModule');
 ```
 
+### `jest.isolateModulesAsync(fn)`
+
+`jest.isolateModulesAsync()` is the equivalent of `jest.isolateModules()`, but for async functions that need to be wrapped. The caller is expected to `await` the completion of `isolateModulesAsync`.
+
+```js
+let myModule;
+await jest.isolateModulesAsync(async () => {
+  constmyModule = require('myModule');
```

Comment:
What about using `await import()` instead of `require()` in this example?

And there is a typo:

```suggestion
  myModule = require('myModule');
```

Reply:
Done in 36c138e2a5cd8796f67a78ae624e5d581ee93d0e